src/test/java/org/owasp/validator/html/test/AntiSammyMalformedPolicy.java

```diff
@@ -0,0 +1,36 @@
+package org.owasp.validator.html.test;
+
+import static org.junit.Assert.fail;
+
+import java.io.InputStream;
+
+import org.junit.Test;
+import org.owasp.validator.html.Policy;
+import org.owasp.validator.html.PolicyException;
+
+public class AntiSammyMalformedPolicy {
```

Comment:
This class should be named: AntiSamyMalformedPolicy. Can you fix that?

Reply:
It also must end in the word "Test" to be picked up by the maven test cycle

`AntiSamyMalformedPolicyTest.java`

Reply:
Done